metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -28,6 +28,14 @@ final class Doctor(
   private val hasProblems = new AtomicBoolean(false)
   private var bspServerName: Option[String] = None
   private var bspServerVersion: Option[String] = None
+  val minimumBloopVersion = "1.3.5"
```

Comment:
I know this was in here before, but is there a way we can just pull this from the version that we're using rather than having it hardcoded? 🤔

Reply:
I moved it to build.sbt. It might actually make sense to tell users to use the same one that we have in the project, since we upgrade the version when we need it. Good idea!

Reply:
Switched to current Bloop version.